tests/integration/suite/daprd/secret/http/fuzz.go

```diff
@@ -61,7 +62,8 @@ func (f *fuzzsecret) Setup(t *testing.T) []framework.Option {
 		for *s == "" ||
 			takenNames[*s] ||
 			len(path.IsValidPathSegmentName(*s)) > 0 ||
-			!reg.MatchString(*s) {
+			!reg.MatchString(*s) ||
+			strings.HasSuffix(*s, ".") {
```

Comment:
It previously worked to have dots as a suffix for both state and secret keys. What's changed that this no longer works?

Reply:
What doesn't work is when a path component is only a dot, such as `state/./.` which happened sometimes in fuzzed tests. This doesn't work because we clean the path using `path.Clean`, which simplifies the dots.

I guess allowing a dot at the end may still work, as long as it's not only a dot. I can change this